.github/workflows/auto-label-issue.yml

```
@@ -0,0 +1,19 @@
+name: Auto Label issues
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
```

Have this run on `windows-latest`

```suggestion
    runs-on: windows-latest
```